packages/x-charts/src/ChartsTooltip/utils.tsx

```diff
@@ -74,13 +75,16 @@ export function useMouseTracker(): UseMouseTrackerReturnValue {
       });
     };
 
+    const throttledHandleMove = throttle(handleMove, 50);
```

Comment:
50ms, so 20 Hz. This feels very lagging on my screen. I think we can't go lower than 120 Hz, so 8ms.

Reply:
And tbh even with no throttling there is usually some base amount of lag, because the OS compositor draws the mouse as the last step just before redrawing the monitor, to make the mouse feel as responsive as possible. Meanwhile apps have access to a less recent mouse position. So anything that doesn't go through the native OS APIs will always have some lag.